app/plot/plotter.py

```diff
@@ -442,9 +527,16 @@ def create_plot(self):
             plot.width_policy = "max"
 
             # Add a label in the center
+            highlight_category = self.config["filters_defaults"][
+                "single_choice_highlight"
+            ]
             for key, value, color in zip(
-                self.raw_data[code][single_choice_key]["x"],
-                self.raw_data[code][single_choice_key]["y"],
+                self.raw_data[code][self.config["filters_defaults"]["single_choice"]][
+                    "x"
+                ],
+                self.raw_data[code][self.config["filters_defaults"]["single_choice"]][
+                    "y"
+                ],
```

Comment:
I don't know why this change was made, this was exactly working before and making the graph more readable. Also here we are re-reading/parsing a dict inside a double loop, instead of reading it once and reusing it afterwards